.github/workflows/heapster.yml

```diff
@@ -0,0 +1,96 @@
+name: SAWScript Heapster
+on:
+  push:
+    branches: [wip-heapster]
+  pull_request:
+    branches: [wip-heapster]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        ghc: ["8.6.5"]
```

Comment:
`8.8.4` and `8.6.5` are currently run on the master branch, so it would make sense to at least add `8.8.4` to make it match. @eddywestbrook would you like to have this additionally enabled?

Reply:
Synced with Eddy on this and he's good with adding more versions, I'll enable that next.